.codacy.yml

```diff
@@ -4,3 +4,4 @@ exclude_paths:
   - 'ci/build_verilator.sh'
   - 'include/vltstd/**'
   - 'nodist/fastcov.py'
+  - 'include/direntmsvc.hpp'
```

Comment:
There are only 2 uses of dirent presently.  I would suggest instead make a separate pull request to #ifdef those to see if a C++17 compiler is available and if so use std::filesystem::directory_entry.   This is somewhat more work but then we don't need the extra header and in a couple years when we require c++17 we can remove the legacy entirely.